pkgs/tools/inputmethods/fcitx5/fcitx5-qt.nix

```
@@ -48,6 +48,12 @@ mkDerivation rec {
     libXdmcp
   ];
 
+  preFixup = ''
+    qtWrapperArgs+=(
+      --prefix FCITX_ADDON_DIRS : $out/lib/fcitx5
+    )
+  '';
```

```suggestion
  qtWrapperArgs = [ "--prefix" "FCITX_ADDON_DIRS" ":" "${placeholder "out"}/lib/fcitx5" ];
```

Should be of the same effect.